### PR TITLE
[models] Removed unused models

### LINKED ISSUE #133: [models] Remove unused models
@rohithasrk  please do this after #110 is merged.

I think we can safely remove the following models (and related base models, admin classes, docs references):

- `RadiusGroup`
- `RadiusGroupUsers`
- `RadiusReply`

These models were meant to implement the features that are being introduce in #110. They copied the same implementation used in [OpenWISP User Management System](https://github.com/openwisp/OpenWISP-User-Management-System) but we ended up in a different path, which I hope will be simpler to grasp/read and maintain.
The actual models aren't really doing anything. They just sit there, it's dead code.

**Please perform a search and replace of these keywords and remove everything.**

### PR DESCRIPTION
**Pull request by rohithasrk**
Implements and closes #133

### CODE REVIEW

File: docs/source/general/abstract_models.rst [hunk 1]
```diff
@@ -1,6 +1,6 @@
-===================
+===============
 Abstract Models
-===================
```
Comment (nemesisdesign) [inline docs/source/general/abstract_models.rst]: thank you so much for this small cleanup.. do more if you spot more

File: docs/source/general/abstract_models.rst [hunk 2]
```diff
@@ -35,25 +35,33 @@ Example:
...
-       group_name = models.CharField(
-               verbose_name=_('groupname'), max_length=255, unique=True, db_column='groupname', db_index=True)
-       priority = models.IntegerField(verbose_name=_('priority'), default=1)
-       creation_date = models.DateField(verbose_name=_('creation date'), null=True, db_column='created_at')
-       modification_date = models.DateField(
-           verbose_name=_('modification date'), null=True, db_column='updated_at')
-       notes = models.CharField(
-           verbose_name=_('notes'), max_length=64, blank=True, null=True)
+    class BaseModel(TimeStampedEditableModel):
+    id = None
```
Comment (nemesisdesign) [inline docs/source/general/abstract_models.rst]: indentation looks wrong here. what do you think?

Comment (rohithasrk) [reply docs/source/general/abstract_models.rst]: Yeah. Will fix. Thanks for pointing out.

File: docs/source/general/abstract_models.rst (hunk 1 above)
Comment (rohithasrk) [reply docs/source/general/abstract_models.rst]: Done with whatever I've found in the latest iteration.

**Comment by rohithasrk** [comment]
Merging this @nemesisdesign.

**Pull request merged**